finagle-thrift/src/main/ruby/lib/finagle-thrift/trace.rb

```diff
@@ -57,6 +57,10 @@ def tracer=(tracer)
     @tracer = tracer
   end
 
+  def stack=(stack)
```

Comment:
My concern is what happens after a ruby server gets a request.  Is it ever legal to have something like:

t1: receives a request
t1: makes a request to a different server
t1: does some different work
t1: puts some work on a different thread, t2
t2: on behalf of t1, makes a request to a different server

... etc.

In this case, they should all logically be part of the same Trace, but if we are storing these as thread-locals, then they won't come along by default when we switch threads.